test/gtest/cfg.cpp

```diff
@@ -133,175 +133,6 @@ TEST_F(CFGTest, CallBlock) {
   EXPECT_EQ(ss.str(), cfgText);
 }
 
-TEST_F(CFGTest, LinearLiveness) {
-  auto moduleText = R"wasm(
-    (module
-      (func $bar
-        (local $a (i32))
-        (local $b (i32))
-        (local $c (i32))
-        (local.set $a
-          (i32.const 1)
-        )
-        (drop
-          (local.get $a)
-        )
-        (local.set $b
-          (local.get $a)
-        )
-        (local.set $c
-          (i32.const 1)
-        )
-        (drop
-          (local.get $c)
-        )
-      )
-    )
-  )wasm";
-
-  auto analyzerText = R"analyzer(CFG Analyzer
-CFG Block: 0
-Input State: 000
-Predecessors:
-Successors:
-Intermediate States (reverse order): 
-000
-block
-000
-drop
-000
-local.get $2
-001
-local.set $2
-000
-i32.const 1
-000
-local.set $1
-000
-local.get $0
-100
-drop
-100
-local.get $0
-100
-local.set $0
-000
-i32.const 1
-000
-End
-)analyzer";
-
-  Module wasm;
-  parseWast(wasm, moduleText);
-
-  CFG cfg = CFG::fromFunction(wasm.getFunction("bar"));
-  size_t numLocals = wasm.getFunction("bar")->getNumLocals();
-
-  FiniteIntPowersetLattice lattice(numLocals);
-  LivenessTransferFunction transferFunction;
-
-  MonotoneCFGAnalyzer<FiniteIntPowersetLattice, LivenessTransferFunction>
```

Comment:
This will be future work. We generally should not have tests that check intermediate results like this because they break when we change internal implementation details. OTOH, the reason the test was like this was that we didn't have a very clean API for accessing the analysis results. I expect to revisit the API problem soon.

(FWIW, generators could help a lot here because we could interleave visiting the expressions and getting their analysis results with test assertions.)